client/src/components/CreateRequest.js

```diff
@@ -28,29 +28,19 @@ export default class extends React.Component {
     }
 
     const performTranslation = async (obj) => {
-      console.log(obj);
-      console.log(ipfs);
-      ipfs.add(new Buffer(obj.string), (err, result) => {
-          if (err) {
-              console.error("Content submission error:", err);
-              return false;
-          } else if (result && result[0] && result[0].hash) {
-              console.log("Successful translation. IPFS address:", result[0].hash);
+      const result = await ipfs.add(new Buffer(obj.string));
```

Comment:
this is much better, but to preserve the error handling capabilities, you might want to wrap this in a try/catch.